shared/common-adapters/usernames.js

```diff
@@ -29,16 +29,16 @@ function usernameText({
       ...(u.you ? globalStyles.italic : null),
     }
 
+    // If onUsernameClicked is omitted, omit the onClick property
+    // entirely, so as to not override any existing onClick handler
+    // from containers on native. (See DESKTOP-3963.)
+    let extraProps = {}
+    if (onUsernameClicked) {
+      const onUsernameClickedNonNull = onUsernameClicked
```

Comment:
Needed to make flow happy (since otherwise `onUsernameClicked` may be set to null before the closure is called)